Questie.lua

```diff
@@ -41,6 +42,14 @@ function Questie:OnDisable()
     end
 end
 
+function Questie:RefreshConfig(db, profile)
+    -- TODO: Migration here?
+    Questie:SetIcons()
+    QuestieQuest:SmoothReset()
+    TrackerBaseFrame:OnProfileChange()
+    print("Switched Profile!")
```

Comment:
Do we want to replace this with a regular `Questie:Print` or remove entirely?

Reply:
Yes!.. there should be _NO_ bare print calls in the code as we want to have the ability to tell em where to print not just have it willy-nilly.. allows better compat with chat addons